### ISSUE TITLE
FE: Dark theme across page?

### ISSUE THREAD

**Issue by maxfenton** [bluecadet/longwood-d8#366]
<img width="923" alt="Screen Shot 2022-02-24 at 4 02 20 PM" src="https://user-images.githubusercontent.com/906502/155610725-9bd905ed-fae3-464b-97ee-e525e7a6f0ad.png">

**Comment by BSBaker2** [comment]
Only implement functionality for entire template

**Comment by maxfenton** [comment]
max: make a branch

**Comment by pingevt** [comment]
@maxfenton makes sense with the header - and prob smarter that way. I'll try and get that toggle in there. If i get that field in, good to push to LIVE then?

**Comment by pingevt** [comment]
@maxfenton - I'm going to try putting this on the .node div. That won't be any performance hit at all compared to trying to put it at the body tag. But if we need to move it up at all, still can do.

**Comment by maxfenton** [comment]
Sounds good
-- 
*Max Fenton (he/him)*
Senior Developer, Bluecadet
PHL 267.639.9956 | NYC 646.360.4252
www.bluecadet.com
